File: Python/4.py
def es_biciesto(año):
    año = int(año)
    if año % 4 == 0 and (año % 100 != 0 or año % 400 == 0):
	    return "Es Bisiesto"
    else:
        return "No es Bisiesto"


print(es_biciesto("2020"))

File: Scala/4.scala
object Bisiesto {
  def main(args: Array[String]): Unit = {
    print("\n________________Bisiesto________________\n")
    print("Introduzca el año: ")
    val año : Int = scala.io.StdIn.readLine.toInt
    print(esBisiesto(año))
  }

  def esBisiesto(x:Int) = {
    if((x%4==0 && x%100 != 0) || x % 400 ==0) "Es Bisiesto" else "No es Bisiesto"
  }
}